applovin_max/ios/Classes/AppLovinMAXNativeAdView.m

```diff
@@ -461,9 +461,9 @@ - (void)sendAdLoadedReactNativeEventForAd:(MANativeAd *)ad
         nativeAdInfo[@"mediaContentAspectRatio"] = @(ad.mediaContentAspectRatio);
     }
     
-    nativeAdInfo[@"isIconImageAvailable"] = @(ad.icon != nil);
-    nativeAdInfo[@"isOptionsViewAvailable"] = @(ad.optionsView != nil);
-    nativeAdInfo[@"isMediaViewAvailable"] = @(ad.mediaView != nil);
+    nativeAdInfo[@"isIconImageAvailable"] = (ad.icon != nil) ? @YES : @NO;
```

Comment:
I'd prefer less code: `@(ad.icon != nil)`

Reply:
I do, but that will be translated into an integer, 1 or 0, on the Flutter side.

Reply:
I see

Reply:
LG then.